components/utility.css

```diff
@@ -88,3 +88,11 @@ h6 {
 .text-align-right {
   text-align: right;
 }
+.selected-component {
+  color: var(--primary-color) !important;
+}
+.selected-component:hover {
+  color: var(--secondary-font-color) !important;
```

Comment:
avoid the use of !important instead what u can do add specificity here

Reply:
Added for this specific one only because I wanted this class to override others.

Reply:
Yeah, avoid using !important otherwise everything is good for merge

Reply:
Rectified it